public/index.html

```diff
@@ -42,6 +42,9 @@
 	<script type="text/javascript">
         window.config = {
             domainName: "https://192.168.88.38",
+            mainColor: "#3596f4",
```

Comment:
in my opinion, you should just store one `theme` object in the config, and merge this with the default theme when calling `createTheme()`. You should not use the color settings from window anywhere else than in the theme.

Reply:
Done